app/src/main/res/layout/settings_page_personal.xml

```diff
@@ -0,0 +1,152 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#FCFCFCFC">
+
+    <Button
+        android:id="@+id/done1"
+        style="?android:attr/borderlessButtonStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="18dp"
+        android:layout_marginStart="310dp"
+        android:fontFamily="@font/roboto_bold"
+        android:text="Done"
+        android:textAllCaps="false"
+        android:textColor="#4F4F4F"
+        android:textSize="24sp" />
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="200dp"
+        android:layout_height="60dp"
+        android:layout_marginStart="104dp"
+        android:layout_marginTop="20dp"
+        android:contentDescription="toggle"
+        app:srcCompat="@drawable/ic_option_pill_bar1" />
+
+    <Button
+        android:id="@+id/toggle1_1"
+        style="?android:attr/borderlessButtonStyle"
+        android:layout_width="60dp"
+        android:layout_height="44dp"
+        android:layout_marginStart="175dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginBottom="2dp"
+        android:backgroundTint="#00FFFFFF"
+        android:visibility="visible" />
+
+    <Button
+        android:id="@+id/toggle1_2"
+        style="?android:attr/borderlessButtonStyle"
+        android:layout_width="60dp"
+        android:layout_height="44dp"
+        android:layout_marginStart="240dp"
+        android:layout_marginTop="22dp"
+        android:backgroundTint="#00FFFFFF"
+        android:visibility="visible" />
+
+    <TextView
+        android:id="@+id/myAccountText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="#282828"
+        android:backgroundTint="#00FFFFFF"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="90dp"
+        android:textSize="20sp"
+        android:text="My account"/>
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_marginTop="115dp"
+        android:background="#000000"/>
+
+    <Button
+        android:id="@+id/myAccountBox"
+        android:layout_width="match_parent"
+        android:layout_height="80dp"
+        android:layout_marginStart="20dp"
+        android:layout_marginEnd="20dp"
+        android:layout_marginTop="135dp"
+        android:backgroundTint="#00FFFFFF"/>
+
+    <ImageView
+        android:id="@+id/calligraphy"
+        android:layout_width="150dp"
+        android:layout_height="49dp"
+        android:layout_marginStart="35dp"
+        android:layout_marginTop="150dp"
+        android:background="#00FFFFFF"
+        android:backgroundTint="#FFFFFF"
+        app:srcCompat="@drawable/ic_calligraphy" />
+
+    <EditText
```

Comment:
the typing bar appears in the center, it should appear on the left. I should also be the same colour as the text